.goreleaser.yaml

```diff
@@ -72,3 +73,46 @@ scoops:
     license: BSD-3
     depends:
       - sops
+
+dockers:
+  - image_templates:
+      - "ghcr.io/budimanjojo/{{.ProjectName}}:v{{.Version}}-amd64"
+    use: buildx
+    dockerfile: Dockerfile
+    build_flag_templates:
+      - "--platform=linux/amd64"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--label=org.opencontainers.image.url=https://github.com/budimanjojo/{{.ProjectName}}"
+      - "--label=org.opencontainers.image.source=https://github.com/budimanjojo/{{.ProjectName}}"
+      - "--label=org.opencontainers.image.licenses=BSD-3-Clause"
+      - "--label=org.opencontainers.image.authors=Budiman Jojo <budimanjojo@gmail.com>"
+      - "--label=org.opencontainers.image.description=A tool to help creating Talos kubernetes cluster"
+  - image_templates:
+      - "ghcr.io/budimanjojo/{{.ProjectName}}:v{{.Version}}-arm64v8"
+    use: buildx
+    goarch: arm64
+    dockerfile: Dockerfile
+    build_flag_templates:
+      - "--platform=linux/amd64"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--label=org.opencontainers.image.url=https://github.com/budimanjojo/{{.ProjectName}}"
+      - "--label=org.opencontainers.image.source=https://github.com/budimanjojo/{{.ProjectName}}"
+      - "--label=org.opencontainers.image.licenses=BSD-3-Clause"
+      - "--label=org.opencontainers.image.authors=Budiman Jojo <budimanjojo@gmail.com>"
+      - "--label=org.opencontainers.image.description=A tool to help creating Talos kubernetes cluster"
+
+docker_manifests:
+  - name_template: "ghcr.io/budimanjojo/{{.ProjectName}}:v{{.Version}}"
+    image_templates:
+      - "ghcr.io/budimanjojo/{{.ProjectName}}:v{{.Version}}-amd64"
+      - "ghcr.io/budimanjojo/{{.ProjectName}}:v{{.Version}}-arm64v8"
+  - name_template: "ghcr.io/budimanjojo/{{.ProjectName}}:latest"
+    image_templates:
```

Comment:
Isn't the `image_templates` here supposed to be `latest` too?

Reply:
It could be, if you want to.

With this setup, you'll basically publish 4 images:
  - image:vX.Y.Z-amd64
  - image:vX.Y.Z:arm64
  - image:vX.Y.Z (multi arch)
  - image:latest (multi arch)

We could also push:
  - image:latest-amd64
  - image:latest-arm64

If you want to

Reply:
Ah, seems like I misunderstood the `image_templates` function. I thought that is a typo. So the `docker_manifests` is just for multi arch images? Ignore me then

Reply:
As far as I understand, yes. The naming is a bit confusing but I think it refers to using `docker manifest` under the hood

Reply:
I can still add per-architecture `latest` tags, if you want to, though!

Reply:
Feel free to do what you deem fit, I'm fine with both.

Reply:
I'll leave it this way for now. I think I'm the only one using the docker image anyway :)))

If people want it later on, it's an easy addition. I think for now this solves the problem it aimed to